OsmAnd/src/net/osmand/plus/views/layers/MapVectorLayer.java

```diff
@@ -25,28 +23,28 @@
 import androidx.annotation.NonNull;
 
 public class MapVectorLayer extends BaseMapLayer {
-
 	private OsmandMapTileView view;
 	private ResourceManager resourceManager;
 	private Paint paintImg;
 
+	private final int VECTOR_LAYER_ID = 1;
```

Comment:
Why 1?
Look for OBF_RASTER_LAYER

Reply:
> Why 1?
Good question, looks like it's typo.
Fixed: `1` => `0`

Renamed to: `VECTOR_LAYER_ID` => `OBF_RASTER_LAYER`